import/popups/PopupOpener.qml

```diff
@@ -69,6 +69,16 @@ Timer {
         return null
     }
 
+    function getCheckboxValue(checkbox) {
+        return (checkbox != null && checkbox != undefined)
```

Comment:
if, else if, else if, else block would work better with this kind of deeply nested ternary operator.

This is type driven checking... Worth adding strict equal comparison.

Reply:
Maybe strictly comparing against false and true for checked.checked and checkbox.value would rendered a bit cleaner block.

```
if (checkbox) {
    // Check for undefined / null
    if (checkbox.checked === true || checkbox.checked == false) {
        return checkbox.checked
    } else if (checkbox.value === true || checkbox.value ===false) {
        return checkbox.value
    }
}

return false
```

or 

```
if (checkbox) {
    // Check for undefined / null
    if (checkbox.checked !== null && checkbox.checked !== undefined) {
        return checkbox.checked
    } else if (checkbox.value !== null && checkbox.value !== undefined) {
        return checkbox.value
    }
}

return false
```

I think decision should be based on the possible checked / value types.

Reply:
Yeah, that's better.

Reply:
Indeed, thanks, done